man/systemd.service.xml

```diff
@@ -1156,6 +1156,35 @@
         kills, this setting determines the state of the unit after <command>systemd-oomd</command> kills a
         cgroup associated with it.</para></listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>OpenFile=</varname></term>
+        <listitem><para>Takes an argument of the form <literal>path:fd-name:options</literal>,
+        where:
+        <itemizedlist>
+            <listitem><simpara><literal>path</literal> is a path to a file or an <constant>AF_UNIX</constant> socket;</simpara></listitem>
```

Comment:
say and AF_UNIX socket *in* *the* *file* *system*.

AF_UNIX also exists in the abstract namespace, hence it's a good idea to be precise here.